CRM/Contact/BAO/Query.php

```diff
@@ -6307,8 +6303,14 @@ public static function processSpecialFormValue(&$formValues, $specialFields, $ch
           // if wildcard is already present return searchString as it is OR append and/or prepend with wildcard
           $isWilcard = strstr($value, '%') ? FALSE : CRM_Core_Config::singleton()->includeWildCardInName;
           $formValues[$element] = ['LIKE' => self::getWildCardedValue($isWilcard, 'LIKE', $value)];
+
         }
       }
+      if (in_array($element, array_keys($changeNames))) {
```

Comment:
@monishdeb we no longer handle renaming fields in the code -in this case we can probably avoid it altogether by adding uniquenames - https://github.com/civicrm/civicrm-core/pull/15652

Reply:
@eileenmcnaughton should we still allow this code and throw a deprecated notice if any of such case arise where unique names were not present?

Reply:
For current purposes we can just leave it unchanged can't we?